src/defaults.js

```diff
@@ -40,7 +40,7 @@ function applyDefaults( passthrough, url, httpMethodName, body, headers, withCre
       return baseUrl;
    }
 
-   return passthrough( httpMethodName || 'GET', modifiedUrl(url, cached), body, headers, withCredentials || false );
+   return passthrough( httpMethodName || 'GET', modifiedUrl(url, cached), body, headers, withCredentials || false, disableBufferCheck || false );
```

Comment:
Though I know you're just copying what was already there, I think `|| false` is a bit of an antipattern since if  `disableBufferCheck` is undefined we could rely on `disableBufferCheck == false`